### ISSUE TITLE
Why does full-screen display mode using mvim on OSX Yosemitie repeat lines of code?

### ISSUE THREAD

**Issue by nwebz** [macvim-dev/macvim#253]
I installed this [vimrc](https://github.com/amix/vimrc) on OSX Yosemite (10.10.5) and every time I enter the full-screen mode with mvim I have a problem that occurs when scrolling down and then back up to the top of the file. The display gets stuck on the top-most line and repeats it in the window. It doesn't actually write to the file, but it is enough to make the application unusable. Does anyone know what the cause may be? I don't even know where to start. 

Here is an example [![example](http://i.stack.imgur.com/Vdjuk.png)](http://i.stack.imgur.com/Vdjuk.png) of my display.

**Comment by malleusinferni** [comment]
I've seen this problem myself in windowed mode, with a custom .vimrc (but several plugins) and on versions of OS X before Yosemite. Don't remember exactly when it started... Probably around the time the linebreak patch was added.

Anyway, I discovered that setting the `lines` variable (from e.g. 54 to 52) would make it go away for at least the lifetime of a window. This is annoying and terrible but since I couldn't pinpoint the source of the problem it was enough to make it usable again.

I have never seen this glitch in the terminal, and in fact I've started favoring the terminal because of it.
